packages/datadog-plugin-azure-functions/src/index.js

```diff
@@ -35,11 +36,19 @@ class AzureFunctionsPlugin extends TracingPlugin {
         'aas.function.trigger': mapTriggerTag(methodName)
       }
     }, false)
+    if (triggerMap[methodName] === 'ServiceBus') {
+      const triggerEntity = invocationContext.options.trigger.queueName || invocationContext.options.trigger.topicName
+      span.setTag('messaging.message_id', invocationContext.triggerMetadata.messageId)
```

Comment:
It would be more efficient to prepare the `meta` object before `startSpan` depending on the trigger, and then pass that directly.

For example:

```js
const meta = getMetaForTrigger(ctx)

const span = this.startSpan(this.operationName(), {
  childOf,
  service: this.serviceName(),
  type: 'serverless',
  meta
}, false)

function getMetaForTrigger({ functionName, methodName, invocationContext }) {
  const meta = {
    'aas.function.name': functionName,
    'aas.function.trigger': mapTriggerTag(methodName)
  }

  if (triggerMap[methodName] === 'ServiceBus') {
    ...
  }

  return meta
}
```